src/anomalib/data/__init__.py

```diff
@@ -28,20 +29,32 @@
 logger = logging.getLogger(__name__)
 
 
-class DataFormat(str, Enum):
-    """Supported Dataset Types."""
+class ImageDataFormat(str, Enum):
+    """Supported Image Dataset Types."""
 
     MVTEC = "mvtec"
     MVTEC_3D = "mvtec_3d"
     BTECH = "btech"
+    KOLEKTOR = "kolektor"
     FOLDER = "folder"
     FOLDER_3D = "folder_3d"
+    VISA = "visa"
+
+
+class VideoDataFormat(str, Enum):
+    """Supported Video Dataset Types."""
+
     UCSDPED = "ucsdped"
     AVENUE = "avenue"
-    VISA = "visa"
     SHANGHAITECH = "shanghaitech"
 
 
+DataFormat = Enum(  # type: ignore[misc]
```

Comment:
Can the type system parse this?

Reply:
Mypy does not complain, but Pylance complains about the following:
```
Variable not allowed in type expression
```